lib/xlsxtream/cell.rb

```diff
@@ -0,0 +1,25 @@
+module Xlsxtream
+  class Cell
+    def initialize(value, bold = false)
```

Comment:
can we use named arguments for styling? So it's self-explanatory. So, `bold: false` as a default